src/index.html

```diff
@@ -118,9 +118,18 @@ <h4 class="data-text border-bottom">Average</h4>
     <section class="data-section grid-item draggable" tabindex="4">
       <h4 class="data-heading">Friends</h4>
       <article class="data-article">
-      <img class="img-medium" src="./images/team.svg" alt="hands in">
-      <p class="subtext" id="friends-step">Weekly Steps</p>
+      <img class="img-medium friends-data" src="./images/team.svg" alt="hands in">
+      <p class="subtext" id="friends-data"></p>
       </article>
+      </article>
+      </article>
+        <section class="friend-icon-container">
+          <button class="friend-hydro-button friend-button">HYDRO</button>
```

Comment:
We should see if this buttons can be tabbed through for accessibility

Reply:
they work with tab